### PR TITLE
InboxItems + InboxItem

### PR DESCRIPTION
**Pull request by seanes**
Adds InboxItems + InboxItem which a 1st iteration stab at the component outlined in Figma. 
I've added a story to Storybook to demonstrate the usage and added it to a separate route (`/inbox`) to demo it in the application.

This PR also includes a couple of rabbit hole tasks in order to make the first story work properly, like including `Inter` fonts.

image

### CODE REVIEW

File: packages/storybook/src/stories/InboxItems/InboxItems.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import type { Meta } from "@storybook/react";
+import { SealIcon, StarIcon } from "@navikt/aksel-icons";
+
+import { InboxItems } from "../../../../frontend-design-poc/src/components/InboxItems";
+import { InboxItem } from "../../../../frontend-design-poc/src/components/InboxItem";
```
Comment (TheKnarf) [inline packages/storybook/src/stories/InboxItems/InboxItems.stories.tsx:6]: Unsure about importing stuff thats outside the package.

Instead one might add `frontend-design-poc` as a dependency of this package and use it that way.

https://pnpm.io/workspaces

I think the dependency becomes something like `workspace:frontend-design-poc` or something similare.

---

Either that or we move the `InboxItems` into the `storybook` package.

File: packages/frontend-design-poc/src/components/InboxItem/InboxItem.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import { Checkbox } from "@digdir/design-system-react";
+import classNames from "classnames";
+
+import styles from "./inboxItem.module.css";
+
+/**
```
Comment (TheKnarf) [inline packages/frontend-design-poc/src/components/InboxItem/InboxItem.tsx:6]: Is VSCode able to pick up this comment when using an `<InboxItem>` component? Or does it have to be just above the component definition (on line 66). I haven't tested this so I'm not sure.

If it works the way it is then this is great.

File: packages/frontend-design-poc/src/components/InboxItems/InboxItems.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ *   <InboxItem {...inboxItemProps3} />
+ * </InboxItems>
+ */
+
+
+export interface InboxItemsProps {
+  children: React.ReactNode;
+}
+
+export const InboxItems = ({ children }: InboxItemsProps) => {
```
Comment (TheKnarf) [inline packages/frontend-design-poc/src/components/InboxItems/InboxItems.tsx:28]: Any reason this is called `InboxItems` and not just `Inbox`?

File: packages/storybook/src/stories/InboxItems/InboxItems.stories.tsx (hunk 1 above)
Comment (seanes) [reply packages/storybook/src/stories/InboxItems/InboxItems.stories.tsx:6]: Yes, I agree ... not very in favor of doing this either. I will have a look into workspaces. Thanks

**Comment by TheKnarf** [comment]
Group `InboxItem` and `InboxItems` into the same folder, so that you can use them as a single import:

```
import { InboxItems, InboxItem } from '../components/InboxItem';
```

Components that are used together should live together.

**Review by alexdigdir — APPROVED** [review]
Bra jobba :D

**Pull request merged**